app.js

```diff
@@ -1,4 +1,54 @@
 //  in this file create an express application - use the middle-ware built into express
 //  to serve up static files from the public directory (index.html and client.js - you 
 //  can also serve up css files from the public directory this way if you want)
-//  you need to support a '/trucks' endpoint, and a dynamic route for '/trucks:name'
\ No newline at end of file
+//  you need to support a '/trucks' endpoint, and a dynamic route for '/trucks:name'
+
+
+
+'use strict';
+
+var trucks = require('./trucks');
+
+var express = require('express');
+var app = express();
+var serveStatic = express.static('public');
+
+app.use(serveStatic);
+
+app.get('/', function (request, response) {
```

Comment:
The express.static middleware has already handled the root route by the time the code gets to this handler, making it unnecessary (not to mention you don't want a redirect here for this application.)